src/borg/algorithms/crc32_dispatch.c

```diff
@@ -75,6 +82,7 @@
 static uint32_t
 crc32_clmul(const uint8_t *src, long len, uint32_t initial_crc)
 {
+    (void)src; (void)len; (void) initial_crc;
```

Comment:
It tells the compiler that these parameters are intentionally not used to avoid warnings about unused parameters. 
I always put these on the same line, because we humans don't really care.